docs/getting-started-guides/environments.md

```diff
@@ -8,4 +8,12 @@ position: 50
 
 !include <environments>
 
+!include <add-new-environments>
+
+:::hint
+Always try to keep the number of environments low, under a dozen or so where possible. Doing so helps with configuring other areas of Octopus, such as Lifecycles, Channels and Security. They also keep your dashboard easy to follow. 
```

Comment:
Re: Lee's comments, probably can trim these sentences down

> In general, keep the number of environments under ten. Having fewer environments makes configuring and maintaining your Octopus Server easier.

Reply:
Fixed in 9954480c9f0e2fcfcff8767f5a63f7bc8af0c970, thanks!